packages/react/src/converter/react-wrapper.tsx

```diff
@@ -44,7 +44,8 @@ export function createReactComponent<BaseComponent extends HTMLElement>(elementN
     }
 
     _propIsReservedReactProp(prop) {
-      const reactProperties = ['children', 'localName', 'ref', 'style'];
+      const reactProperties = ['children', 'localName', 'ref', 'style', 'className'];
+      // const reactProperties = ['children', 'localName', 'ref', 'style'];
```

Comment:
Is this an extra comment?

Reply:
Oops, my bad, `git add -p` gone wrong! I removed it, still not sure what to do about the test case however.

Reply:
No worries. Thanks for the quick fix 🥇